tests/unit/configFile.spec.js

```diff
@@ -133,4 +133,239 @@ describe("configFile.js", () => {
       expect(result).toStrictEqual(changedSettings);
     });
   });
+
+  describe("parseBasicSettings function", () => {
+    it("data should be parsed correctly", async () => {
+      await LocalStore.clear();
+      const expectedParsedData = {
+        configurationURL:
+          "https://raw.githubusercontent.com/AdvancedThreatAnalytics/threat-analytics-search/main/settings.json",
+        useGroups: true,
+        configEncrypted: false,
+        configEncryptionKey: "",
+        autoUpdateConfig: false,
+      };
+      const result = await ConfigFile.default.parseBasicSettings(
+        defaultSettings.config
+      );
+
+      expect(result).toEqual(expect.objectContaining(expectedParsedData));
+      expect(result.mergeGroups).toEqual(result.useGroups);
+    });
+  });
+
+  describe("parseGroups function", () => {
+    it("only array of length 3 are parsed, result should be an array of object with first two being enabled", async () => {
+      const groups = [
+        ["1", "IP Lookup"],
+        ["2", "Domain"],
+        ["3", "Hash"],
+      ];
+      const expectedResult = [
+        { name: "IP Lookup", enabled: true },
+        { name: "Domain", enabled: true },
+        { name: "Hash", enabled: false },
+      ];
+      var result = await ConfigFile.default.parseGroups(groups.slice(0, 1));
+      expect(result).toHaveLength(0);
+
+      result = await ConfigFile.default.parseGroups(groups);
+      expect(result).toHaveLength(3);
+      expect(result).toStrictEqual(expectedResult);
+    });
```

Comment:
Please split this into two separate tests: `it("Array of length < 3 isn't parsed")` and `it(Array of length >= 3 are parsed and only first two items are enabled")`